cmd/convox/rack.go

```diff
@@ -69,6 +70,10 @@ func init() {
 						Name:  "stats",
 						Usage: "display process cpu/memory stats",
 					},
+					cli.BoolFlag{
+						Name:  "a, all",
```

Comment:
I think `-a` is OK here since it's under the `convox rack` namespace. `ps -a` feels like a natural command to me.